core/arch/arm/kernel/core_bootcfg.c

```diff
@@ -169,7 +169,7 @@ static struct map_area bootcfg_memory_map[] = {
 };
 
 /*
- * bootcfg_get_pbuf_is_handler - return the platform specfic pbuf_is
+ * bootcfg_get_pbuf_is_handler - return the platform specific pbuf_is
```

Comment:
file core_bootcfg.c renamed in core_common_bootcfg.c ?  Same below on entry_a32.S and entry_a64.S.